example/index.html

```diff
@@ -0,0 +1,171 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Halogen Datepicker Example</title>
+    <style>
+
+body {
+  font-family: sans-serif;
+  margin: auto;
+}
+
+.ColorPicker {
```

Comment:
It could be, but I thought that, if someone wants to use the lib, they will still change visuals, use their own convention for class names or some preprocessors, atc.

So basicly this styles is not part of the component. It's just an example which one could copy, modify and evolve independent of the component.
(also because of layout and not specifying any class names in PS component it's very easy to do achieve that)

That you think on it? 

/cc @toastal that's  your thoughts on this?